tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -331,11 +331,11 @@ binary path.
 $ sudo apt-get install python-numpy swig python-dev
 ```
 
-#### Configure the installation {#configure}
+#### Configure the installation 
 
 Run the `configure` script at the root of the tree.  The configure script
 asks you for the path to your python interpreter and allows (optional)
-configuration of the CUDA libraries (see [below](#configure_cuda)).
+configuration of the CUDA libraries (see [below](#configure-tensorflow-s-canonical-view-of-cuda-libraries)).
```

Comment:
Noticed that apostrophes aren't handled correctly. This should be `#configure-tensorflows-canonical-view-of-cuda-libraries`

I'll patch this in